ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -201,6 +209,10 @@ export const specificPopulations = [
   }
 ];
 
+const billingDescription = <div>The <i>All of Us</i> Program provides $200 in free credits per user.
```

Comment:
Noting we'll need to keep this in sync with the actual value. Probably fine for now

Reply:
Yes, default makes sense. It would also be very confusing to explain which user this is referring to (since it depends on the creator, not necesarily the current user's free tier credits).